### PR TITLE
Global TN Search Guide

### PR DESCRIPTION
**Pull request by jchavez443**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tns-guide.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tns-guide.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tns-guide.s3-website-us-east-1.amazonaws.com/index.html

**Review by jmulford-bw — APPROVED** [review]

File: numbers/guides/globalTnSearch.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+* [Retrieving TN Lata](#retrieving-tn-lata)
+* [Retrieving TN Reservation](#retrieving-tn-reservation)
+* [Retrieving TN Available Nn Routes](#retrieving-tn-availableNnRoutes)
+
+## About {#about}
+
+The Bandwidth Telephone Numbers API allows you to search for telephone numbers across all of your accounts that your user credentials have access to.  You can then view detailed information about the telephone numbers on any of your accounts.
+
+### Note
+> When querying for a phone number that is NOT in your account you will recieve an error `Telephone Numbers cannot be found on accounts`.  Within an `<ErrorList>` XML list.
```
Comment (hamermike) [inline numbers/guides/globalTnSearch.md]:
```suggestion
> When querying for a phone number that is NOT in your account you will receive an error `Telephone Numbers cannot be found on accounts`.  Within an `<ErrorList>` XML list.
```

File: numbers/guides/globalTnSearch.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+            <LastModified>2016-11-16T19:34:22.000Z</LastModified>
+        </TelephoneNumber>
+    </TelephoneNumbers>
+</TelephoneNumbersResponse>
+```
+{% endextendmethod %}
+
+{% extendmethod %}
+## Retrieving TN Information {#retrieving-tn-information}
+Retrieves information about the specified telephone number. The information returned provides status and historic information about the Telephone Number, including the status, the order id and date associated with the last modification, and the account and site information associated with the TN.  The request for more information can be made by requesting a number of specific derived sub-resources.
```
Comment (hamermike) [inline numbers/guides/globalTnSearch.md]:
```suggestion
Retrieves information about the specified telephone number. The information returned provides status and historic information about the Telephone Number, including the status, the order id and date associated with the last modification, and the account and site information associated with the TN. The request for more information can be made by requesting a number of specific derived sub-resources.
```

**Review by hamermike — CHANGES_REQUESTED** [review]
Tiny change.

**Review by ajrice6713 — APPROVED** [review]

File: numbers/guides/globalTnSearch.md (hunk 2 above)
Comment (jchavez443) [reply numbers/guides/globalTnSearch.md]: How did you even find that! Haha

**Pull request merged**